corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -614,15 +615,35 @@ def _forms_with_attachments(domain, app_id, xmlns, datespan, user_types):
             pass
 
 
+def _get_forms_with_attachments(es_query):
+    query = es_query.source(['_id', 'external_blobs'])
+
+    for form in query.scroll():
+        try:
+            for attachment in form.get('external_blobs', {}).values():
+                if attachment['content_type'] != "text/xml":
+                    yield form
+                    continue
+        except AttributeError:
+            pass
+
+
+# ToDo: Remove post build_form_multimedia_zipfile rollout
 def get_form_ids_having_multimedia(domain, app_id, xmlns, datespan, user_types):
     return {
         form['_id'] for form in _forms_with_attachments(domain, app_id, xmlns, datespan, user_types)
     }
 
 
-def media_export_is_too_big(domain, app_id, xmlns, datespan, user_types):
+def get_form_ids_with_multimedia(export_es_query):
```

Comment:
deprecates get_form_ids_having_multimedia above.
The difference being just the method called now is `_get_forms_with_attachments` instead of `_forms_with_attachments`

Reply:
Same as previous comment.